client/client-multi/client-c/deploy.sh

```diff
@@ -0,0 +1,14 @@
+#! /bin/bash
```

Comment:
`travis` runs scripts in bash.
Furthermore, argument array slicing is a `bash` feature, not supported by `sh`.